hibike/devices/ExampleDevice/example_device.cpp

```diff
@@ -38,8 +38,8 @@ void loop() {
   // so exessive delays here will affect hibike.
   //don't change Kumiko.
 
-  hazuki += 0;
-  sapphire += 1;
+  // hazuki += 0;
```

Comment:
Not sure how I feel about leaving it this way.

Espescially since we've decided that running hibike_process.py as a tester is deprecated.